packages/manager/apps/web/client/app/private-database/database/ram/update/private-database-database-ram-update.html

```diff
@@ -46,55 +46,25 @@
         <div
             data-wizard-step
-            data-wizard-step-on-load="getDurations"
-            data-wizard-step-valid="ramCtrl.model.duration"
+            data-wizard-step-on-load="getProvisionalOrder"
+            data-wizard-step-valid="ramCtrl.data.provisionalOrder"
         >
-            <p data-translate="privateDatabase_RAM_add_select_duration"></p>
+            <p data-translate="privateDatabase_RAM_add_tarification"></p>
 
-            <div class="text-center" data-ng-if="ramCtrl.loading.durations">
+            <div
+                class="text-center"
+                data-ng-if="ramCtrl.loading.provisionalOrder"
+            >
                 <oui-spinner></oui-spinner>
                 <span data-translate="privateDatabase_RAM_loading_order"></span>
             </div>
-
-            <form name="orderRamForm" data-ng-if="!ramCtrl.loading.durations">
-                <div
-                    class="oui-radio"
-                    data-ng-repeat="duration in (ramCtrl.data.durations.available | orderBy:'duration':false) track by $index"
-                >
-                    <input
-                        type="radio"
-                        class="oui-radio__input"
-                        id="ramDuration-{{$index}}"
-                        name="ramDuration"
-                        data-ng-model="ramCtrl.model.duration"
-                        data-ng-value="duration"
-                    />
-                    <label
-                        class="oui-radio__label-container"
-                        for="ramDuration-{{$index}}"
-                    >
-                        <span class="oui-radio__label">
-                            <span class="oui-radio__icon"></span>
-                            <span class="oui-radio__text">
-                                <span
-                                    data-ng-if="ramCtrl.constructor.isProrataDuration(duration)"
-                                    data-ng-bind-html="duration.duration | wucDuration:'longDate':{prorata:true}"
-                                >
-                                </span>
-                                <span
-                                    data-ng-if="!ramCtrl.constructor.isProrataDuration(duration)"
-                                    data-ng-bind-html="duration.duration | wucDuration:'longDate'"
-                                >
-                                </span
-                                >&nbsp;:
-                                <span
-                                    data-ng-bind-html="duration.prices | price: ramCtrl.data.ovhSubsidiary"
-                                ></span>
-                            </span>
-                        </span>
-                    </label>
-                </div>
-            </form>
+            <div data-ng-if="!ramCtrl.loading.provisionalOrder">
+                <span data-translate="privateDatabase_RAM_add_duration"> </span>
+                &nbsp;:
```

Comment:
Prefer the use of utility classes
```suggestion
                <span class="pr-2" data-translate="privateDatabase_RAM_add_duration"></span>
```